### ISSUE TITLE
nil is not a legal json value

### ISSUE THREAD

**Issue by bremoran** [cabo/cddlc#2]
The output of cddlc includes `nil`, which is not a legal value in JSON. The correct output value for these instances is `null`.

**Comment by cabo** [comment]
Indeed, if you don't specify an output format, you get Ruby format.
For JSON output  use -tjson (or -tneat, which selects a specific JSON formatter -- I might add others later).
For CDDL output use -tcddl.
